### ISSUE TITLE
Unable to load plugin , windows 10. Latest nightly mta version. (not x64)

### ISSUE THREAD

**Issue by LaNataliaaa** [qaisjp/mta-amx#68]
**Describe the bug**
Have tried to load plugin for windows, but keeps telling me server was unable to load the plugin king.dll. I have it in deathmatch/modules folder.

**To reproduce**

**Expected behaviour**
To load the plugin

**Screenshots**

**Version**
Latest from nightly builds

**Additional context**

**Comment by Disinterpreter** [comment]
Yes. Thank you for your issue. I guess the problem depends on #55. You can fix it if you download sqlite3.dll and put it into the server folder like: 
![image](https://user-images.githubusercontent.com/3816155/87761674-8be58080-c81a-11ea-8e7e-1c3a79911002.png)
